From: John Paul Adrian Glaubitz <glaubitz@physik.fu-berlin.de>
Date: Sat, 29 Dec 2018 01:57:00 +0100
Subject: Strip -g -O2 from CGAL_CXX_FLAGS_INIT

We need to strip -g -O2 from CGAL_CXX_FLAGS_INIT so we can
build the tests on mips and mipsel with -g1 to reduce the
debug info and therefore memory footprint.

--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -524,6 +524,8 @@
   string(REPLACE "FORTIFY_SOURCE=2" "FORTIFY_SOURCE=0" CGAL_CXX_FLAGS_INIT ${CGAL_CXX_FLAGS_INIT})
 endif()
 
+string(REPLACE "-g -O2" "" CGAL_CXX_FLAGS_INIT ${CGAL_CXX_FLAGS_INIT})
+
 # GLib2
 
 find_package(GLIB2 2.2.0 REQUIRED)
